packages/web/src/lib/web-mode.ts

```diff
@@ -38,7 +41,7 @@ export class WebMode {
 				pendingInfo[CONFIRMED_ROUND] !== null &&
```

Comment:
How about only waiting 10 or 6 round?
We defined [WAIT_ROUNDS](https://github.com/scale-it/algo-builder/blob/develop/packages/web/src/lib/constants.ts#L4) in constants.ts